doc/src/manual/performance-tips.md

```diff
@@ -48,79 +54,79 @@ so all the performance issues discussed previously apply.
 
-As a teaser, an improved version of this function allocates no memory
+If we instead pass `x` as an argument to the function it no longer allocates memory
 (the allocation reported below is due to running the `@time` macro in global scope)
-and has an order of magnitude faster execution after the first call:
+and is significantly faster after the first call:
 
 ```julia-repl
-julia> @time f_improved(1)
-  0.007008 seconds (1.32 k allocations: 63.640 KiB)
-0.5
+julia> @time sum_arg(x)
+  0.007701 seconds (821 allocations: 43.059 KiB)
+496.84883432553846
 
-julia> @time f_improved(10^6)
-  0.002997 seconds (6 allocations: 192 bytes)
-2.5000025e11
+julia> @time sum_arg(x)
+  0.000006 seconds (5 allocations: 176 bytes)
+496.84883432553846
 ```
 
-Below you'll learn how to spot the problem with `f` and how to fix it.
-
 In some situations, your function may need to allocate memory as part of its operation, and this
 can complicate the simple picture above. In such cases, consider using one of the [tools](@ref tools)
 below to diagnose problems, or write a version of your function that separates allocation from
 its algorithmic aspects (see [Pre-allocating outputs](@ref)).
 
+!!! note
+    For more serious benchmarking, consider the [BenchmarkTools.jl](https://github.com/JuliaCI/BenchmarkTools.jl)
+    package which among other things evaluates the function multiple times in order to reduce noise.
+
 ## [Tools](@id tools)
 
 Julia and its package ecosystem includes tools that may help you diagnose problems and improve
 the performance of your code:
 
   * [Profiling](@ref) allows you to measure the performance of your running code and identify lines
-    that serve as bottlenecks.  For complex projects, the [ProfileView](https://github.com/timholy/ProfileView.jl)
+    that serve as bottlenecks. For complex projects, the [ProfileView](https://github.com/timholy/ProfileView.jl)
     package can help you visualize your profiling results.
   * Unexpectedly-large memory allocations--as reported by [`@time`](@ref), [`@allocated`](@ref), or
     the profiler (through calls to the garbage-collection routines)--hint that there might be issues
-    with your code.  If you don't see another reason for the allocations, suspect a type problem.
+    with your code. If you don't see another reason for the allocations, suspect a type problem.
      You can also start Julia with the `--track-allocation=user` option and examine the resulting
-    `*.mem` files to see information about where those allocations occur.  See [Memory allocation analysis](@ref).
+    `*.mem` files to see information about where those allocations occur. See [Memory allocation analysis](@ref).
   * `@code_warntype` generates a representation of your code that can be helpful in finding expressions
     that result in type uncertainty. See [`@code_warntype`](@ref) below.
-  * The [Lint](https://github.com/tonyhffong/Lint.jl)
```

Comment:
I don't think this is maintained enough to be linked to.

Reply:
Is Traceur.jl ready to be linked here?

Reply:
Added